.gitignore

```diff
@@ -7,6 +7,9 @@ pids
 *.pid
 *.seed
 
+# Configuration for integration tests
+test/config.json
```

Comment:
What's the concern here? If people actually use said file for testing there changes, this helps them not include it in a pull request. Telling people that such file is used for testing is not really that significant - it's obvious reading the tests. If you suggest that we leave the tests out altogether I see this point, but don't necessarily agree with excluding tests. If there are solid arguments to do so happy to reconsider :)